2주차/코딩도장-13.7-심사문제.py

```diff
@@ -0,0 +1,13 @@
+'''
+표준 입력으로 가격(정수)과 쿠폰 이름이 각 줄에 입력됩니다.
+ Cash3000 쿠폰은 3,000원, Cash5000 쿠폰은 5,000원을 할인합니다. 
+쿠폰에 따라 할인된 가격을 출력하는 프로그램을 만드세요.
+(input에서 안내 문자열은 출력하지 않아야 합니다)
+'''
+
+p = int(input())
+c = input()
```

Comment:
변수명은 딱 봤을 때 어떤 것을 의미하는지 알 수 있도록 지어야 합니다.
```suggestion
price = int(input())
coupon = input()
```